src/test/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/integration/pages/IntegrationTestDataFactory.kt

```diff
@@ -34,124 +34,151 @@ private val bonnieElizabeth =
     gender = "female"
   )
 
-fun standardMoveM4() =
-  Move(
-    moveId = "M4",
-    updatedAt = LocalDateTime.now(),
-    supplier = Supplier.GEOAMEY,
-    moveType = MoveType.STANDARD,
-    status = MoveStatus.completed,
-    reference = "STANDARDMOVE2",
-    fromNomisAgencyId = "PRISON3",
-    fromSiteName = "PRISON THREE",
-    toNomisAgencyId = "PRISON4",
-    toSiteName = "PRISON FOUR",
-    reportFromLocationType = "prison",
-    reportToLocationType = "prison",
-    pickUpDateTime = LocalDateTime.of(2020, 12, 1, 10, 20),
-    dropOffOrCancelledDateTime = LocalDateTime.of(2020, 12, 1, 12, 20),
-    person = Person(
-      personId = "_",
+object Dec2020MoveData {
```

Comment:
I grouped the move data accordingly so it is clear what period of time it represents.